tokenLists/terra2.tokenlist.json

```diff
@@ -141,6 +141,20 @@
       "token": "terra1s50rr0vz05xmmkz5wnc2kqkjq5ldwjrdv4sqzf983pzfxuj7jgsq4ehcu2",
       "icon": "../img/CW/cw.svg",
       "decimals": 6
+    },
```

Comment:
those 2 token contracts don't exist on terra 2 testnet, they should be added under "phoenix-1" and not "pisco-1"